scripts/sysman/device.yml

```diff
@@ -43,6 +43,11 @@ desc: "Maximum number of characters in string properties."
 name: $S_STRING_PROPERTY_SIZE
 value: "64"
 --- #--------------------------------------------------------------------------
+type: macro
+desc: "Maximum device universal unique id (UUID) size in bytes."
+name: $S_MAX_DEVICE_UUID_SIZE
```

Comment:
Recommend renaming `$S_MAX_DEVICE_UUID_SIZE` to `$S_MAX_UUID_SIZE` as it would be a more general name.